code/modules/holiday/nth_week.dm

```diff
@@ -88,6 +88,7 @@
 /datum/holiday/nth_week/ice_cream
 	name = ICE_CREAM_DAY
 	begin_week = 3
+	begin_month = JULY
```

Comment:
i disagree lets go back to every third sunday

Reply:
Only if oranges agrees.